src/ast.rs

```diff
@@ -8,6 +8,15 @@ pub struct Call {
 pub enum OpName {
     Plus,
     Minus,
+    Multiply,
+    Divide,
+}
+
+#[derive(Debug)]
+pub struct BinaryOp {
+    pub(crate) operation: OpName,
+    pub(crate) left_hand_side: Box<Expression>,
```

Comment:
might `lhs` and `rhs` be acceptable mathematical abbreviations in this context?

Reply:
I'll go for `left` and `right` and maintain my dislike for acronyms DTSGTY?